test/methods.jl

```diff
@@ -45,6 +48,20 @@ end
     @test_throws ErrorException flat_correct(ones(5, 5), ones(5, 5), norm_value = -2)
 end
 
+@testset "trim" begin
+    @test trim(ones(5, 5), (:, 4:5)) == trim!(ones(5, 5), (:, 4:5))
+    @test trim(ones(5, 5), (4:5, :)) == trim!(ones(5, 5), (4:5, :))
```

Comment:
Note that these tests are not particularly useful: you're basically testing that the trimmed arrays on both sides have the same shape, not the same content, since they're all ones anyway.  You should be using a different array to test with, for example something like `reshape(1:25, 5, 5)` or `rand(5, 5)`

Reply:
Sure! I will modify the tests.